### PR TITLE
added timestamp to SubscriptionWrapper for storage in Mongo, defaulte…

### PR DESCRIPTION
**Pull request by mattrobertsky**
…d to now()

Please review.

### CODE REVIEW

File: app/uk/gov/hmrc/softdrinksindustrylevy/services/MongoBufferService.scala [hunk 1]
```diff
@@ -26,7 +27,7 @@ import uk.gov.hmrc.softdrinksindustrylevy.models.json.internal._
 class MongoBufferService @Inject()(implicit mc: MongoConnector)
   extends ReactiveRepository[SubscriptionWrapper, String]("sdil-subscription", mc.db, SubscriptionWrapper.format, implicitly)
 
-case class SubscriptionWrapper(_id: String, subscription: Subscription)
+case class SubscriptionWrapper(_id: String, subscription: Subscription, timestamp: LocalDateTime = LocalDateTime.now())
```
Comment (adamdye-hmrc) [inline app/uk/gov/hmrc/softdrinksindustrylevy/services/MongoBufferService.scala]: Do we need these braces?

Comment (mattrobertsky) [reply app/uk/gov/hmrc/softdrinksindustrylevy/services/MongoBufferService.scala]: No. Just testing you.

File: app/uk/gov/hmrc/softdrinksindustrylevy/services/MongoBufferService.scala [hunk 2]
```diff
@@ -26,10 +27,10 @@ import uk.gov.hmrc.softdrinksindustrylevy.models.json.internal._
...
   extends ReactiveRepository[SubscriptionWrapper, String]("sdil-subscription", mc.db, SubscriptionWrapper.format, implicitly)
 
-case class SubscriptionWrapper(_id: String, subscription: Subscription)
+case class SubscriptionWrapper(_id: String, subscription: Subscription, timestamp: LocalDateTime = LocalDateTime.now)
 
 object SubscriptionWrapper {
   implicit val subFormat: Format[Subscription] = Format(subReads, subWrites)
 
   val format: Format[SubscriptionWrapper] = Json.format[SubscriptionWrapper]
-}
```
Comment (adamdye-hmrc) [inline app/uk/gov/hmrc/softdrinksindustrylevy/services/MongoBufferService.scala]: Why have you added a line?

Comment (mattrobertsky) [reply app/uk/gov/hmrc/softdrinksindustrylevy/services/MongoBufferService.scala]: we haven't

Comment (adamdye-hmrc) [reply app/uk/gov/hmrc/softdrinksindustrylevy/services/MongoBufferService.scala]: touché

**Pull request merged**